src/swap/SwapExecuteScreen.tsx

```diff
@@ -31,18 +32,59 @@ export function SwapExecuteScreen() {
   }
 
   const navigateToSupport = () => {
-    replace(Screens.SupportContact)
+    navigate(Screens.SupportContact)
   }
 
-  const NavigateHomeIcon = () => {
-    return (
-      <View style={styles.navigateHomeIcon}>
-        <TopBarIconButton icon={<Times />} testID="ReturnHome" onPress={navigateHome} />
-      </View>
-    )
-  }
+  const navigationButtons = useMemo(() => {
+    switch (swapState) {
+      case SwapState.ERROR:
+        return (
+          <View style={styles.actionBar}>
+            <Button
+              text={t('SwapExecuteScreen.swapActionBar.tryAgain')}
+              onPress={navigateToReviewScreen}
+              type={BtnTypes.PRIMARY}
+              size={BtnSizes.FULL}
+              testID="SwapExecuteScreen/TryAgain"
+              style={[styles.button, { paddingBottom: Spacing.Regular16 }]}
+            />
+            <Button
+              text={t('SwapExecuteScreen.swapActionBar.done')}
+              onPress={navigateHome}
+              type={BtnTypes.SECONDARY}
+              size={BtnSizes.FULL}
+              testID="SwapExecuteScreen/Done"
+              style={styles.button}
+            />
+          </View>
+        )
+      case SwapState.COMPLETE:
+        return (
+          <View style={styles.actionBar}>
+            <Button
+              text={t('SwapExecuteScreen.swapActionBar.done')}
+              onPress={navigateHome}
+              type={BtnTypes.PRIMARY}
+              size={BtnSizes.FULL}
+              testID="SwapExecuteScreen/Done"
+              style={[styles.button, { paddingBottom: Spacing.Regular16 }]}
```

Comment:
the paddingBottom style already exists in the button style so it can be removed here :)